thoth/storages/graph/models.py

```diff
@@ -61,6 +61,19 @@ class PackageExtractRun(VertexBase):
     os_version_id = model_property(type=str, index="exact")
 
 
+@attr.s(slots=True)
+class PackageAnalyzerRun(VertexBase):
+    """A class representing a single package-analyzer (package analysis) run."""
+
+    package_analysis_document_id = model_property(type=str, index="exact")
+    package_analysis_datetime = model_property(type=datetime, index="hour")
+    package_analyzer_version = model_property(type=str, index="exact")
+    package_analyzer_name = model_property(type=str, index="exact")
+    debug = model_property(type=bool)
+    package_analyzer_error = model_property(type=bool, index="exact")
```

Comment:
**package_analyzer_error** is queryable.

Reply:
I think the correct configuration to index `bool` types is `index="bool"`.